runtime/src/main/java/dev/ebullient/micrometer/runtime/CompositeRegistryCreator.java

```diff
@@ -2,28 +2,18 @@
 
 import static javax.interceptor.Interceptor.Priority.PLATFORM_AFTER;
 
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.Set;
-
-import javax.enterprise.inject.Any;
 import javax.enterprise.inject.Produces;
-import javax.enterprise.inject.spi.Bean;
-import javax.enterprise.inject.spi.BeanManager;
-import javax.enterprise.inject.spi.CDI;
 import javax.inject.Singleton;
 
 import org.jboss.logging.Logger;
 
-import io.micrometer.core.instrument.Clock;
 import io.micrometer.core.instrument.MeterRegistry;
 import io.micrometer.core.instrument.composite.CompositeMeterRegistry;
 import io.quarkus.arc.AlternativePriority;
 
 /**
  * Ensure there is one resolvable MeterRegistry.
  * <p>
- * This is invoked/created by MicrometerProcessor#createRootRegistry.
  * Fetch MeterRegistry instances using types discovered from registered Producer fields and methods.
  * If there are no instances, create a no-op/empty CompositeMeterRegistry. If there are several instances,
  * create a CompositeMeterRegistry that aggregates the collection. Otherwise, return the single discovered
```

Comment:
Nice to have: dump the comment?

Reply:
I removed because it's not called by that anymore

Reply:
No .. I mean.. remove the whole comment.. I think you've nixed all the code it explains. ;)

Reply:
Doh, yeah